syntax = "proto3"; // 声明了protobuf的版本

package protoMsg; // 声明了代码所在的包（对于C++来说是namespace）

//请求登录的响应
message LoginResponse
{
    string uname = 1;
    string id = 2;
}
